src/postgraphql/schema/auth/__tests__/getPgTokenTypeFromIdentifier-test.js

```diff
@@ -0,0 +1,39 @@
+import withPgClient from '../../../../__tests__/utils/withPgClient'
+import introspectDatabase from '../../../../postgres/introspection/introspectDatabase'
+import getPgTokenTypeFromIdentifier from '../getPgTokenTypeFromIdentifier'
+
+// good names start with a letter or underscore and may also contain numbers after position 0
+const goodNames = [
+  'test', 'test_0', 'test_1', 'test_2', 'test_3', 'test_4', 'test_5', 'test_6', 'test_7', 'test_8', 'test_9', '_test',
```

Comment:
This only really needs to test edge cases, so 'test_0', 'test_9' and '_test' should suffice